src/laddie/laddie_utilities.f90

```diff
@@ -0,0 +1,208 @@
+MODULE laddie_utilities
+
+  ! Utilities for the laddie model
+
+! ===== Preamble =====
+! ====================
+    
+  USE precisions                                             , ONLY: dp
+  USE mpi_basic                                              , ONLY: par, sync
+  USE control_resources_and_error_messaging                  , ONLY: crash, init_routine, finalise_routine, colour_string
+  USE model_configuration                                    , ONLY: C
+  USE parameters
+  USE mesh_types                                             , ONLY: type_mesh
+  USE ice_model_types                                        , ONLY: type_ice_model
+  USE laddie_model_types                                     , ONLY: type_laddie_model
+  USE ocean_model_types                                      , ONLY: type_ocean_model
+  USE reallocate_mod                                         , ONLY: reallocate_bounds
+  USE ocean_utilities                                        , ONLY: interpolate_ocean_depth
+
+  IMPLICIT NONE
+    
+CONTAINS
+    
+! ===== Main routines =====
+! =========================
+
+  SUBROUTINE compute_ambient_TS( mesh, laddie, ocean, ice)
+    ! Initialise the laddie model
```

Comment:
I don't think that's what this routine does

Reply:
Correct, will modify